src/dmd/traits.d

```diff
@@ -672,6 +684,14 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
 
         return isFuncX(f => !f.needThis() && !f.isNested());
     }
+    if (e.ident == Id.isModule)
+    {
```

Comment:
Add a 
```
        if (dim != 1)
            return dimError(1);
```

to get 
```
enum C2 = __traits(isPackage); // TODO: needs to fail
enum D2 = __traits(isModule);
```
To fail.

Reply:
Yeah; I probably should have figured that one out faster, given that basically every other `__traits` conditional in the file does that. But it's one of those days. ;)